python/labours/labours.py

```diff
@@ -15,22 +12,16 @@
 import threading
 import time
 import warnings
+from collections import defaultdict, namedtuple
+from datetime import datetime, timedelta
+from importlib import import_module
+from itertools import chain
```

Comment:
Sorry, missed this during the first pass: our standard imports sorting is "appnexus", which does not differentiate between regular and "from" records. It should have been checked in the CI, but it hasn't, apparently.
https://github.com/src-d/hercules/issues/315

Reply:
Check. This was just the default PyCharm "Optimize Imports" order :)